### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by arthurvandijk**
Initial set up for the main rubicon branch containing repo guidelines and templates.

### CODE REVIEW

File: README.md
Comment (jpaarhuis) [inline README.md]: typo: mathcing

File: README.md
Comment (jpaarhuis) [inline README.md]: Combine these two bullet points, Almost seems as it require reviews from all Code Owners, while it only require one review from any of the Code Owners.

File: README.md
Comment (jpaarhuis) [inline README.md]: Nice work on the templates! Are you also planning on (maybe optionally) include testing steps?

**Review by jpaarhuis — CHANGES_REQUESTED** [review]

**Comment by jpaarhuis** [comment]
Volgens mij moet hier nog wat gebeuren.

File: README.md
Comment (arthurvandijk) [reply README.md]: Adding (optional) testing steps is planned and will be added in the next release. 
The dotnet core and Angular templates will be updated with default (optional) testing steps

**Review by jpaarhuis — APPROVED** [review]

**Pull request merged**
